src/ontology/mondo-edit.obo

```diff
@@ -349224,15 +349217,24 @@ replaced_by: MONDO:0006045
 [Term]
 id: MONDO:0018368
 name: primary peritoneal serous/papillary carcinoma
-comment: Editor note: TODO merge with MONDO:0003817 peritoneal serous papillary adenocarcinoma
 subset: ordo_disease {source="Orphanet:398980"}
+synonym: "peritoneal serous papillary adenocarcinoma" EXACT [MONDO:0003817]
 synonym: "PPSPC" EXACT [Orphanet:398980]
+synonym: "primary peritoneal serous papillary adenocarcinoma" EXACT [NCIT:C7695]
+synonym: "primary peritoneal serous papillary carcinoma" EXACT [NCIT:C7695]
+synonym: "primary serous papillary carcinoma of peritoneum" EXACT [DOID:6228, NCIT:C7695]
+xref: DOID:6228 {source="MONDO:equivalentTo"}
 xref: ICD10:C48.0 {source="Orphanet:398980", source="ORDO:398980/ntbt"}
 xref: ICD10:C48.1 {source="Orphanet:398980", source="ORDO:398980/ntbt"}
 xref: ICD10:C48.2 {source="Orphanet:398980", source="ORDO:398980/ntbt"}
 xref: ICD10:C48.8 {source="Orphanet:398980", source="ORDO:398980/ntbt"}
+xref: NCIT:C7695 {source="MONDO:kboom-pr-0.92/0.83/0.18", source="DOID:6228", source="MONDO:equivalentTo"}
 xref: Orphanet:398980 {source="MONDO:equivalentTo"}
+xref: SCTID:15674004 {source="DOID:6228"}
+xref: UMLS:C1368918 {source="NCIT:C7695", source="DOID:6228", source="MONDO:equivalentTo"}
 xref: UMLS:CN205035 {source="MEDGEN:kboom-pr98-c99", source="MONDO:equivalentTo"}
+is_a: MONDO:0003195 {source="DOID:6228"} ! peritoneal serous adenocarcinoma
+is_a: MONDO:0006386 {source="NCIT:C7695"} ! primary peritoneal serous adenocarcinoma
 is_a: MONDO:0018364 {source="Orphanet:398980"} ! malignant epithelial tumor of ovary
```

Comment:
are we sure?

Reply:
looks like this is outdated, I don't see 'peritoneal serous adenocarcinoma' or 'primary peritoneal serous adenocarcinoma' as synonyms for this term.